incubator/fluentd-cloudwatch/templates/daemonset.yaml

```diff
@@ -20,6 +20,15 @@ spec:
 {{- end }}
     spec:
       serviceAccountName: {{ if .Values.rbac.create }}{{ template "fluentd-cloudwatch.fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
+      initContainers:
+        - name: copy-fluentd-config
+          image: busybox
+          command: ['sh', '-c', 'cp /config-volume/* /etc/fluentd']
```

Comment:
I think it needs to be /config-volume/..data/* (qwerk of how configmap atomic updates happen)

Reply:
@kfox1111 You lgtm suggests this can be merged despite this comment. How to proceed?